File: echoReceiver.java
import java.net.*;
import java.io.*;

public class echoReceiver {
    public static void main(String[] args) throws Exception {
        Socket socket = new Socket("localhost", 1234);

        BufferedReader userInput = new BufferedReader(
                new InputStreamReader(System.in));
        BufferedReader in = new BufferedReader(
                new InputStreamReader(socket.getInputStream()));
        PrintWriter out = new PrintWriter(socket.getOutputStream(), true);

        System.out.print("Enter message: ");
        String msg = userInput.readLine();
        out.println(msg);

        System.out.println("Echo from server: " + in.readLine());

        socket.close();
    }
}

File: Echo Receiver.py
import socket

s = socket.socket()
s.connect(("localhost", 7000))
print("Connected to server")

while True:
    msg = input("Client: ")
    s.send(msg.encode())
    if msg == "exit":
        break
    print("Server:", s.recv(1024).decode())

s.close()
